### DISCUSSION TITLE
404

### DISCUSSION THREAD

**Discussion by uckhanmalek** [iranxray/hope#83] [General]
سلام. بعد از یک روز کار کردن برای ورود به پنل ارور ۴۰۴ میده
در صورتی که پنل در حال کار هست و کانفیگها وصل هستن

**Comment by saeidghodrati** [comment]
منم همین مشکل دارم

**Comment by mhd200548** [comment]
روی مرورگر موبایل باز میشه
ولی ویندوز خطای 404 میده

**Comment by Phoenix-999** [comment]
@mhd200548 @saeidghodrati 

تنظیمات DNS تغییر دهید مشکل شما حال خواهد شد 

راهنمای تنظیمات شکن
https://shecan.ir/tutorials/

**Comment by odkeihancpu** [comment]
سلام. چطور dns را تغییر دهم ؟ 
تنظیمات dns کلاینت یا سرور؟

**Comment by Phoenix-999** [comment]
@odkeihancpu سلام
لینکی که فرستادم را دنبال کنید اگر مشکلی بود اسکرین شات بگیرید تا راهنمای کنم 

راهنمای تنظیمات شکن
https://shecan.ir/tutorials/
